test/testinput/bufr_ncep_mtiasi.yaml

```diff
@@ -20,6 +20,72 @@ observations:
+            type: floatVector
+            value: [ 64875.0, 65200.0, 65275.0, 65350.0, 65425.0, 65500.0, 65575.0, 65650.0, 65700.0, 65725.0,
+                     65750.0, 65800.0, 65850.0, 65875.0, 65900.0, 65950.0, 66000.0, 66025.0, 66050.0, 66125.0,
+                     66175.0, 66225.0, 66275.0, 66325.0, 66375.0, 66425.0, 66450.0, 66500.0, 66525.0, 66550.0,
+                     66575.0, 66600.0, 66625.0, 66650.0, 66700.0, 66775.0, 66800.0, 66850.0, 66900.0, 66950.0,
+                     67000.0, 67050.0, 67075.0, 67125.0, 67200.0, 67225.0, 67250.0, 67300.0, 67375.0, 67450.0,
+                     67525.0, 67600.0, 67675.0, 67750.0, 67800.0, 67850.0, 67925.0, 68000.0, 68075.0, 68125.0,
+                     68175.0, 68225.0, 68250.0, 68325.0, 68400.0, 68450.0, 68475.0, 68500.0, 68550.0, 68650.0,
+                     68725.0, 68800.0, 68875.0, 68950.0, 68975.0, 69100.0, 69150.0, 69250.0, 69300.0, 69400.0,
+                     69450.0, 69475.0, 69525.0, 69550.0, 69600.0, 69650.0, 69725.0, 69775.0, 69800.0, 69825.0,
+                     69900.0, 69925.0, 69950.0, 70025.0, 70075.0, 70100.0, 70125.0, 70175.0, 70225.0, 70250.0,
+                     70275.0, 70375.0, 70400.0, 70450.0, 70550.0, 70625.0, 70700.0, 70775.0, 70825.0, 70950.0,
+                     70975.0, 71025.0, 71100.0, 71150.0, 71200.0, 71350.0, 71450.0, 71525.0, 71600.0, 71825.0,
+                     71875.0, 71950.0, 71975.0, 72050.0, 72125.0, 72200.0, 72300.0, 72475.0, 72550.0, 72625.0,
+                     72650.0, 72700.0, 72775.0, 72850.0, 73100.0, 73150.0, 73225.0, 73325.0, 73375.0, 73475.0,
+                     73550.0, 73625.0, 73750.0, 73775.0, 73800.0, 73850.0, 73900.0, 73950.0, 74000.0, 74050.0,
+                     74125.0, 74200.0, 74425.0, 74500.0, 74575.0, 74600.0, 74650.0, 74675.0, 74725.0, 74750.0,
+                     74825.0, 74875.0, 74925.0, 75050.0, 75125.0, 75175.0, 75275.0, 75300.0, 75325.0, 75450.0,
+                     75525.0, 75600.0, 75725.0, 75900.0, 75950.0, 76275.0, 76400.0, 76550.0, 77200.0, 77350.0,
+                     78125.0, 78275.0, 78450.0, 78625.0, 78750.0, 78800.0, 78925.0, 79075.0, 79325.0, 80100.0,
+                     80625.0, 81025.0, 81175.0, 82100.0, 82950.0, 83375.0, 84400.0, 86150.0, 87125.0, 87500.0,
+                     90150.0, 90625.0, 91725.0, 91925.0, 92500.0, 92800.0, 93800.0, 94250.0, 94325.0, 95025.0,
+                     96250.0, 96550.0, 97925.0, 99700.0, 99825.0, 99975.0, 100075.0, 100150.0, 100225.0, 100325.0,
+                     100475.0, 100525.0, 100600.0, 100725.0, 100825.0, 100975.0, 101050.0, 101200.0, 101325.0, 101450.0,
+                     101550.0, 101650.0, 101825.0, 101875.0, 102025.0, 102100.0, 102200.0, 102225.0, 102300.0, 102425.0,
+                     102500.0, 102625.0, 102700.0, 102775.0, 102875.0, 102900.0, 103000.0, 103100.0, 103175.0, 103300.0,
+                     103475.0, 103675.0, 103825.0, 103950.0, 104050.0, 104100.0, 104150.0, 104625.0, 105125.0, 105450.0,
```

Comment:
Needs to go under `globals` in the ioda section.

Are these values going to be the same for all IASIA files?

Reply:
I think we were talking about an extension that could get values from a text file in the future, but better bet right now might be to use python to read the file instead. Please see the following examples:

https://github.com/JCSDA-internal/ioda-converters/blob/develop/test/testinput/bufr_query_python_to_ioda_test.py
https://github.com/JCSDA-internal/ioda/blob/develop/src/engines/test/ioda/python/test_ioda_obsspace.py

Reply:
these values are the same for all `IASI` MetOp-A, -B and -C

Reply:
Cool! So this should work as long as it's moved into the ioda::globals section of the YAML configuration file.